app/src/main/java/com/chartiq/chartiqsample/studies/StudyOptionsActivity.java

```diff
@@ -260,30 +351,16 @@ public void onClick(View v) {
         });
         if ("auto".equals(parameter.color)) {
             color.setBackgroundColor(Color.BLACK);
-        } else {
-            try{
-                if(parameter.color.contains("rgb")) {
-                    String subString = parameter.color.substring(parameter.color.indexOf('(') + 1, parameter.color.indexOf(')'));
-                    String rgbColors[] = subString.split(",");
-                    int parsedColor = 0;
-                    int alphaValue = 255;
-
-                    // contains an alpha value
-                    if(rgbColors.length == 4) {
-                        double value = Double.parseDouble(rgbColors[3].trim());
-                        value = Math.floor(value >= 1.0 ? 255 : value * 256.0); // use 256 for floating point precision when value is less than 1.0
-                        alphaValue = (int) value;
-                    }
-                    parsedColor = Color.argb(alphaValue, Integer.parseInt(rgbColors[0].trim()),
-                            Integer.parseInt(rgbColors[1].trim()), Integer.parseInt(rgbColors[2].trim()));
-
-                    color.setBackgroundColor(parsedColor);
-                } else {
-                    color.setBackgroundColor(Color.parseColor(parameter.color));
-                }
-            }catch(Exception e) {
-                e.printStackTrace();
+        } else if(parameter.color != null){
+            color.setBackgroundColor(Color.parseColor(parameter.color));
```

Comment:
I'm should assume that `Color.parseColor` will handle the rgb values that the removed code once did?

Reply:
This is odd... I branched off master but the changes for rgb committed last month aren't there. Something is wrong.